src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -0,0 +1,54 @@
+package com.kodilla.ecommercee.domain;
+
+import javax.persistence.*;
+
+@Entity(name = "cart")
+public class Cart {
```

Comment:
Encja nie jest potrzebna - będzie osobne zadanie

Reply:
Wiem, że z zasad CleanCode niepotrzebny kod nie powinien zaśmiecać, ale czy konieczne jest jego skasowanie, jeśli wiemy na 100%, że potem będzie wykorzystany?

Reply:
W tym momencie klasa powinna zostać skasowana, to bedzie kolejne zadanie